src/hooks/useInterval.ts

```diff
@@ -1,19 +1,19 @@
 import { useCallback, useEffect, useRef } from 'react';
 
 export default function useInterval() {
-  const handle = useRef();
+  const handle = useRef<number>();
 
   const start = useCallback((func, ms) => {
     handle.current = setInterval(func, ms);
   }, []);
 
   const stop = useCallback(
-    () => handle.current && clearInterval(handle.current),
+    () => (handle.current ? clearInterval(handle.current) : undefined),
```

Comment:
out of curiosity, what was this change for?

Reply:
TS yelled at me.